### ISSUE TITLE
accuracy = sum(correct)/n_ground_truth : ZeroDivision Error: Division by Zero

### ISSUE THREAD

**Issue by sebyo** [kenshohara/3D-ResNets-PyTorch#279]
I got this error while trying to evaluate top-1 video accuracy of a recognition result.I used the same script provided 
python -m util_scripts.eval_accuracy ~/data/kinetics.json ~/data/results/val.json --subset val -k 1 --ignore
 does anyone have any idea how to solve this problem ( from my understanding it's unable to load the groundtruth) ?
ps: for the training I used the UCF101.

**Comment by 9bxlue** [comment]
hello there,i yet meet the same questions,please tell me how do you solve?

**Comment by sebyo** [comment]
@9bxlue just in the --subset change it to validation not val
